packages/iTwinUI-react/src/core/UserIcon/UserIcon.tsx

```diff
@@ -52,6 +64,8 @@ export const defaultStatusTitles: StatusTitles = {
 };
 
 /**
+ * @deprecated Since v2, this has been renamed to `Avatar`
```

Comment:
This is not the way to do it, because now the component code is duplicated for no reason.

Here's how I would do it:
0. Rename all instances of `UserIcon` and `UserIconGroup` to `Avatar` and `AvatarGroup`.
1. Delete every trace of `UserIcon` and `UserIconGroup`. This includes test files and test images.
2. In `Avatar`, add `export const UserIcon = Avatar`. In `AvatarGroup`, add `export const AvatarGroup = UserIconGroup`. Same thing with props. And then add `@deperacted` JSDoc above these.
3. In the barrel file, export all four components and their props.

Reply:
**[DONE]** Good idea, thanks.

To confirm, is the old JSDocs needed for `UserIcon` and `UserIconGroup`? Or is the `@deprecation` tag enough? ([L128](https://github.com/iTwin/iTwinUI-react/pull/902/files#diff-316029e888ce93252e0623c6960c4df4d157b8aa6403db243a65a030da39345aR128) and [L144](https://github.com/iTwin/iTwinUI-react/pull/902/files#diff-b55ac7cab532d4e79274a4b578b5348be2369f627f66d51da118b7a3ef3ec2c9R144)) I felt the `@deprecation` tag was enough since users should be encouraged to not use our deprecated components/props.

Reply:
deprecation should be enough